libs/ledger-live-common/src/families/solana/types.ts

```diff
@@ -212,10 +213,12 @@ export type SolanaStakeWithMeta = {
 
 export type SolanaResources = {
   stakes: SolanaStake[];
+  unstakeReserve: BigNumber;
 };
 
 export type SolanaResourcesRaw = {
   stakes: string;
+  unstakeReserve: BigNumber;
```

Comment:
`*Raw` types must only contain native types (i.e compatible with `JSON.stringify`)

Please use `string` here and update the functions in `serialization.ts`

Reply:
Sure! Fixed in [9ef14bc](https://github.com/LedgerHQ/ledger-live/pull/5228/commits/9ef14bcbe9bb1741142803b1df8bfb96fd4d7b01). Just curious, what is the purpose for this serialization?

Reply:
Account is saved in the cache, this method transform the actual "living" object in storable data, we want only native types there